docs/agents/sematext-agent/releasenotes.md

```diff
@@ -39,6 +39,22 @@ NULL
 
 -->
 
+## Version 3.7.0
+
+Date: September 23, 2024
+
+### New Features
+- **Windows Events**: We are happy to share that Sematext Agent now supports collecting Windows Events. You can define which event logs to collect and send to Sematext Cloud, refer to the [configuration documentation](https://sematext.com/docs/integration/configuring-windows-agent/).
```

Comment:
Suggestion: 
Sematext Agent now supports collecting Windows Events. You can define which event logs to collect and send them to Sematext Cloud. See [Configuring the Sematext Windows Agent](https://sematext.com/docs/integration/configuring-windows-agent/) for more details.